driver-testsuite/tests/Basic/ContentTest.php

```diff
@@ -21,6 +21,18 @@ public function testOuterHtml()
         );
     }
 
+    public function testDumpingEmptyElements()
+    {
+        $this->getSession()->visit($this->pathTo('/index.html'));
+
+        $element = $this->getAssertSession()->elementExists('css', '#empty');
+
+        $this->assertEquals(
+            "An empty <em></em> tag should be rendered with both open and close tags.",
```

Comment:
I think this might fail if some drivers preserve the leading and trailing spaces in the HTML, as you have some of them in the HTML